app/services/idv/steps/verify_base_step.rb

```diff
@@ -2,8 +2,8 @@ module Idv
   module Steps
     class VerifyBaseStep < DocAuthBaseStep
       AAMVA_SUPPORTED_JURISDICTIONS = %w[
-        AR AZ CO CT DC DE FL GA IA ID IL IN KS KY MA MD ME MI MO MS MT NC ND NE
-        NJ NM OH OR PA RI SC SD TN TX VA VT WA WI WY
+        AL AR AZ CA CO CT DC DE FL GA HI IA ID IL IN KS KY LA MA MD ME MI MN MO MS MT NC ND NE
+        NJ NM NV NY OH OR PA RI SC SD TN TX UT VA VT WA WI WY
```

Comment:
this includes some big states (like CA, NY), are we worried about capacity? should we move this to a config value so we could update the list quickly we needed to?

Reply:
> should we move this to a config value so we could update the list quickly we needed to?

We should, yes. It's also an AC of the story.

Reply:
Might be good to lean on the cautious side... @aduth  what do you think re above?